### PR TITLE
moved collateral to additional panel

### PR DESCRIPTION
**Pull request by smartdevcom**
Peek 2023-01-11 00-58

### CODE REVIEW

**Comment by peetzweg** [comment]
Super cool, love the gifs as well. 👍

File: packages/ui/components/pages/PoolPage/MarketsList/CollateralModal/PendingTransaction.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const PendingTransaction = ({
             All Done!
           </Text>
           <Text variant="mdText" fontWeight="bold">
-            You {asset.membership ? 'disabled' : 'enabled'} {asset.underlyingSymbol} as collateral
```
Comment (peetzweg) [inline packages/ui/components/pages/PoolPage/MarketsList/CollateralModal/PendingTransaction.tsx:36]: Oh this was wrong? Good catch.

**Comment by peetzweg** [comment]
Maybe the `Collateral.tsx` file should be moved as it no longer belongs to the MarketList components?
Not sure where we keep the Funding Operation Components or if there are any separate ones.

**Review by peetzweg — APPROVED** [review]

**Comment by smartdevcom** [comment]
Rearranged components inside `MarketList` component

**Pull request merged**